.github/container/manifest.yaml

```diff
@@ -93,54 +93,54 @@ jestimator:
 optax:
   url: https://github.com/google-deepmind/optax.git
   tracking_ref: main
-  latest_verified_commit: 623609c7a77a19d48b021cbc300262308846317e
+  latest_verified_commit: 18110a40153097d15015af2d5d2ee73a86c2a9df
   mode: pip-vcs
 seqio:
   url: https://github.com/google/seqio.git
   tracking_ref: main
-  latest_verified_commit: e31af8c1a11f749edeac512f34d148b9933f863f
+  latest_verified_commit: 11706e4a1e01a81ea6b3e02c5ad147028d5b94bb
   mode: pip-vcs
 # used by Pallas
 openxla-triton:
   url: https://github.com/openxla/triton.git
   tracking_ref: llvm-head
-  latest_verified_commit: cl608559313
+  latest_verified_commit: c33bbb01d087129e3714cfc2a0bcd53b80dd2013
```

Comment:
What is the workflow that resulted in this change?
The `openxla-triton` commit is derived from the `xla` commit.
In this case, the old value is still valid: https://github.com/openxla/xla/blob/e025d9f3ed4d5b7ed80b2352c613e3706d917416/third_party/triton/workspace.bzl#L8
The `bump.sh` script should get this right:
https://github.com/NVIDIA/JAX-Toolbox/blob/9c2ae8fcde0cb2e0d14273ed0107634df0cbc5a7/.github/container/bump.sh#L114-L120
so I guess these changes came from something else?

Reply:
I did use `bump.sh` to update the manifest. My command was simply
```
bash bump.sh -i manifest.yaml
```